modules/ossm-cert-manager-integration.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration_{context}"]
+== Integrating cert-manager with Service Mesh
```

Comment:
```suggestion
== Integrating Service Mesh with cert-manager and istio-csr
```
It's crucial to emphasize that we provide integration with **cert-manager/istio-csr**, not only with cert-manager.